.flake8

```diff
@@ -1,5 +1,6 @@
 [flake8]
 #E266 too many leading '#' for block comment
 #E722 do not use bare 'except'
-ignore = E266,E722

```

Comment:
why are you changing this

Reply:
Adding W503 to the ignore list.  W503 requires that multi-line conditionals have the boolean dangling on the prior line where you have to go find it.  The new, preferred syntax is to have the booleans at the beginning of the continued line.  

See lines 82 plus or minus of firmware_policy.py (I'll add a comment) which W503 did not like where I put the boolean operators at the beginning of continued lines, not the end of prior lines.  I agree with Knuth, PEP8, and the community at large who said that this prior anti-pattern should actually be the recommended pattern (i.e. W503 should be ignored, consider requiring W504).

PEP8:  https://github.com/python/peps/commit/c59c4376ad233a62ca4b3a6060c81368bd21e85b
https://github.com/PyCQA/pycodestyle/issues/513
https://github.com/python-diamond/Diamond/commit/b62bc3fe91a138613c464846f1650797c1df6f9c